src/include/dns-server/dialog-masterzone.rb

```diff
@@ -207,37 +200,37 @@ def GetMasterZoneEditorTabBasics
       deep_copy(contents)
     end
 
-    def ZoneAclInit
-      allowed = false
-      keys = []
-      Builtins.foreach(Ops.get_list(@current_zone, "options", [])) do |m|
-        if Ops.get_string(m, "key", "") == "allow-transfer" && !allowed
-          key = Builtins.regexpsub(
-            Ops.get_string(m, "value", ""),
-            "^.*\\{[ \t]*(.*)[ \t]*\\}.*$",
-            "\\1"
-          )
+    # @return [Array<String>]
+    def current_zone_allow_transfer
+      items = []
+      @current_zone.fetch("options", []).each do |m|
```

Comment:
I wasn't able to find explicit definition / initialization of ```@current_zone```. Is it an intention?